android/src/main/java/com/reactlibrary/RNMailCoreModule.java

```diff
@@ -27,6 +27,7 @@
   public RNMailCoreModule(ReactApplicationContext reactContext) {
     super(reactContext);
     this.reactContext = reactContext;
+    this.mailClient = new MailClient();
```

Comment:
field declaration for `mailClient` is missing. BasicExample does not build.
Could you please check?
also, can you add test code for the new methods in BasicExample ?

Reply:
add support for all methods in ios, if you could check it it would be great